src/main/java/main/Main.java

```diff
@@ -17,63 +21,67 @@ public static void main(String[] args) {
-        employeeSet.forEach(System.out::println);
-        System.out.println();
+        Logger.getLogger().info("Original created HashSet: " + " size " + employeeSet.size());
+        employeeSet.forEach(Logger.getLogger()::info);
+        Logger.getLogger().info("\n");
 
         /**
          * Sorting HashSet using Collections class
          */
         Set<Employee> sortingUsingCollectionsClass = sortEmployee.sortUsingCollections(employeeSet);
-        System.out.println("Sorting HashSet using Collections class: " + " size " + sortingUsingCollectionsClass.size());
-        sortingUsingCollectionsClass.forEach(System.out::println);
-        System.out.println();
+        Logger.getLogger().info("Sorting HashSet using Collections class: " + " size " + sortingUsingCollectionsClass.size());
+        sortingUsingCollectionsClass.forEach(Logger.getLogger()::info);
+        Logger.getLogger().info("\n");
 
         /**
          * Sorting HashSet using Collections class with Comparator
          */
         Set<Employee> sortingUsingCollectionsWithComparator = sortEmployee.sortUsingComparatorAndCollections(employeeSet, comparatorProcessor.getComparator(ComparatorType.LAST_NAME_SORTED));
-        System.out.println("Sorting HashSet using Collections class with Comparator: " + " size " + sortingUsingCollectionsWithComparator.size());
-        sortingUsingCollectionsWithComparator.forEach(System.out::println);
-        System.out.println();
+        Logger.getLogger().info("Sorting HashSet using Collections class with Comparator: " + " size " + sortingUsingCollectionsWithComparator.size());
+        sortingUsingCollectionsWithComparator.forEach(Logger.getLogger()::info);
+        Logger.getLogger().info("\n");
 
         /**
          * Sorting HashSet using StreamAPI
          */
         Set<Employee> sortingUsingStreamAPI = sortEmployee.sortUsingStream(employeeSet);
-        System.out.println("Sorting HashSet using Stream API: " + " size " + sortingUsingStreamAPI.size());
-        sortingUsingStreamAPI.forEach(System.out::println);
-        System.out.println();
+        Logger.getLogger().info("Sorting HashSet using Stream API: " + " size " + sortingUsingStreamAPI.size());
+        sortingUsingStreamAPI.forEach(Logger.getLogger()::info);
+        Logger.getLogger().info("\n");
 
         /**
          * Sorting HashSet using StreamAPI and Comparator
          */
         Set<Employee> sortingUsingStreamWithComparator = sortEmployee.sortUsingComparatorAndStream(employeeSet, comparatorProcessor.getComparator(ComparatorType.LAST_NAME_SORTED));
-        System.out.println("Sorting HashSet using Stream API and Comparator: " + " size " + sortingUsingStreamWithComparator.size());
-        sortingUsingStreamWithComparator.forEach(System.out::println);
-        System.out.println();
+        Logger.getLogger().info("Sorting HashSet using Stream API and Comparator: " + " size " + sortingUsingStreamWithComparator.size());
```

Comment:
It would be better to define info, error ect methods in your Logger class and call them directly from Logger class  (ex. Logger.info or info (using static import))
And please make sure you've added outpot files to git ignore